libr/cons/pal.c

```diff
@@ -118,7 +111,8 @@ R_API void r_cons_pal_init () {
 	cons->cpal.args               = (RColor) RColor_YELLOW;
 	cons->cpal.bin                = (RColor) RColor_CYAN;
 	cons->cpal.btext              = (RColor) RColor_YELLOW;
-	cons->cpal.call               = (RColor) RColor_BGREEN;
+	cons->cpal.call               = (RColor) RColor_GREEN;
+	cons->cpal.call.attr          = 1u << 1;
```

Comment:
Why don't you use a `#define`? Something like RCOLOR_BOLD or RCOLOR_ATTR_BOLD?

Reply:
Or an enum

Reply:
It is not a color so I don't use `RCOLOR_*`

Reply:
Yes but imho having an enum is better than having to guess what 1u << 3 means.